### PR TITLE
Updates README.md to use correct name for package/container type

### PR DESCRIPTION
**Pull request by nweiler**
Updated to use correct name for package/container type.

### CODE REVIEW

**Comment by cwille97** [comment]
@nweiler Thank you for your contribution. This tool does support [containers](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry) so the README is correct.

**Comment by nweiler** [comment]
Ok. My usage suggests that the tool does in fact support containers but that the tag 'containers' does not work and 'docker' does.

**Comment by nweiler** [comment]
@cwille97 For example, here is the output from two separate GitHub actions runs. I have redacted some sensitive info. AFAIK the only difference between the two runs is the name change:

## SUCCESS
```
Run actions/delete-package-versions@v4
  with:
    owner: XXXXXXXX
    package-name: XXXXXXXX
    package-type: docker
    num-old-versions-to-delete: 10
    delete-only-pre-release-versions: false
    delete-only-untagged-versions: false
    token: ***
    min-versions-to-keep: -1
    ignore-versions: ^$
Total versions deleted till now: 10
```

## FAIL
```
Run actions/delete-package-versions@v4
  with:
    owner: XXXXXXXX
    package-name: XXXXXXXX
    package-type: container
    num-old-versions-to-delete: 10
    delete-only-pre-release-versions: false
    delete-only-untagged-versions: false
    token: ***
    min-versions-to-keep: -1
    ignore-versions: ^$
Error: get versions API failed. Package not found.
```

**Comment by kalinon** [comment]
`package-type: container` works for me in v5. Try the new version.

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ This action deletes versions of a package from [GitHub Packages](https://github.
   # Required
   package-name:
 
-  # Type of the package. Can be one of container, maven, npm, nuget, or rubygems.
+  # Type of the package. Can be one of docker, maven, npm, nuget, or rubygems.
```
Comment (cwille97) [inline README.md:35]:
```suggestion
  # Type of the package. Can be one of docker (v4 or older), container (v5 or newer), maven, npm, nuget, or rubygems.
```

**Review by cwille97 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nweiler) [reply README.md:35]: Beautiful! Thank you for your input.

**Review by cwille97 — APPROVED** [review]
LGTM, thanks for the docs update!

**Pull request merged**